%\section{Research Papers}
\section{Publications}
\vspace*{-.7cm}
%
%Journals
%Refereed
%Conference Proceedings
%\begin{enumerate}
\begin{outerlist}

\item
\textbf{Sergey Nurk}$^*$, Sergey Koren$^*$, Arang Rhie$^*$, Mikko Rautiainen$^*$ ($^*$equal contribution)%et al.
, The Telomere-to-Telomere Consortium, Evan Eichler, Karen Miga, Adam Phillippy\\
\textbf{``The complete sequence of a human genome''.}\\
Submitted, bioRxiv, 2021

\item
Mitchell Vollger, Xavi Guitart, Philip Dishuck, Ludovica Mercuri, William Harvey, Ariel Gershman, Mark Diekhans, Arvis Sulovari, Katherine Munson, Alexandra Lewis, Kendra Hoekzema, David Porubsky, Ruiyang Li, \textbf{Sergey Nurk}, Sergey Koren, Karen Miga, Adam Phillippy, Winston Timp, Mario Ventura, Evan Eichler\\
\textbf{``Segmental duplications and their variation in a complete human genome''.}\\
Submitted, bioRxiv, 2021

\item
Tiago Leão, Mingxun Wang, Nathan Moss, Ricardo da Silva, Jon Sanders, \textbf{Sergey Nurk}, Alexey Gurevich, Gregory Humphrey, Raphael Reher, Qiyun Zhu, Pedro Belda-Ferre, Evgenia Glukhov, Syrena Whitner, Kelsey Alexander, Robert Rex, Pavel Pevzner, Pieter C Dorrestein, Rob Knight, Nuno Bandeira, William H Gerwick, Lena Gerwick\\
\textbf{``A Multi-Omics Characterization of the Natural Product Potential of Tropical Filamentous Marine Cyanobacteria''.}\\
Marine Drugs, Jan;19(1), 2021

\item
Glennis Logsdon, Mitchell Vollger, PingHsun Hsieh, Yafei Mao, Mikhail Liskovykh, Sergey Koren, \textbf{Sergey Nurk}, Ludovica Mercuri, Philip Dishuck, Arang Rhie, Leonardo de Lima, David Porubsky, Andrey Bzikadze, Milinn Kremitzki, Tina Graves-Lindsay, Chirag Jain, Kendra Hoekzema, Shwetha Murali, Katherine Munson, Carl Baker, Melanie Sorenson, Alexandra Lewis, Urvashi Surti, Jennifer Gerton, Vladimir Larionov, Mario Ventura, Karen Miga, Adam Phillippy, Evan Eichler\\
\textbf{``The structure, function, and evolution of a complete human chromosome 8''.}\\
Nature, May;593(7857):101-107, 2021

\item
Christopher Quince, \textbf{Sergey Nurk}, Sebastien Raguideau, Robert James, Orkun Soyer, J Kimberley Summers, Antoine Limasset, A Murat Eren, Rayan Chikhi, Aaron Darling\\
\textbf{``Metagenomics Strain Resolution on Assembly Graphs''.}\\
In press., bioRxiv, 2020

\item
\textbf{Sergey Nurk}, Brian Walenz, Arang Rhie, Mitchell Vollger, Glennis Logsdon, Robert Grothe, Karen Miga, Evan Eichler, Adam Phillippy and Sergey Koren\\
\textbf{``HiCanu: accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads''.}\\
Genome Research, Sep;30(9):1291-1305, 2020

\item
Tatiana Dvorkina, Dmitry Antipov, Anton Korobeynikov and \textbf{Sergey Nurk}\\
\textbf{``SPAligner: alignment of long diverged molecular sequences to assembly graphs''.}\\
BMC Bioinformatics, 21(Suppl 12), 2020

\item
Jon G. Sanders, \textbf{Sergey Nurk}, Rodolfo A. Salido, Jeremiah Minich, Zhenjiang Z. Xu, Qiyun Zhu, Cameron Martino, Marcus Fedarko, Timothy D. Arthur, Feng Chen, Brigid S. Boland, Greg C. Humphrey, Caitriona Brennan, Karenina Sanders, James Gaffney, Kristen Jepsen, Mahdieh Khosroheidari, Cliff Green, Marlon Liyanage, Jason W. Dang, Vanessa V. Phelan, Robert A. Quinn, Anton Bankevich, John T. Chang, Tariq M. Rana, Douglas J. Conrad, William J. Sandborn, Larry Smarr, Pieter C. Dorrestein, Pavel A. Pevzner and Rob Knight\\
\textbf{``Optimizing sequencing protocols for leaderboard metagenomics by combining long and short reads''.}\\
Genome Biology, 20, 226, 2019

\item
Xin Fang, Jonathan Monk, \textbf{Sergey Nurk}, Margarita Akseshina, Qiyun Zhu, Christopher Gemmell, Connor Gianetto-Hill, Nelly Leung, Richard Szubin, Jon Sanders, Paul L Beck, Weizhong Li, William J Sandborn, Scott D Gray-Owen, Rob Knight, Emma Allen-Vercoe, Bernhard O Palsson, Larry Smarr\\
\textbf{``Metagenomics-based, strain-level analysis of Escherichia coli from a time-series of microbiome samples from a Crohn’s disease patient''.}\\
Frontiers in Microbiology 9, 2559, 2018

\item
\textbf{Sergey Nurk}$^*$, Dmitry Meleshko$^*$ ($^*$equal contribution), Anton Korobeynikov, and Pavel A. Pevzner \\
\textbf{``metaSPAdes: a new versatile metagenomic assembler.''} \\
Genome Research, 27(5), 824-834, 2017

\item
Yu Lin, \textbf{Sergey Nurk} and Pavel A. Pevzner\\
\textbf{``What is the difference between the breakpoint graph and the de Bruijn graph?''.} \\
%Proceedings of the Twelfth Annual Research in Computational Molecular Biology (RECOMB) Satellite Workshop on Comparative Genomics
BMC genomics, 15(6), S6, 2014

\item Gabriela I Guzman, Jose Utrilla, \textbf{Sergey Nurk}, Elizabeth Brunk, Jonathan M Monk, Ali Ebrahim, Bernhard Ø Palsson, Adam M. Feist\\
\textbf{``Model-driven discovery of underground metabolic functions in Escherichia coli''.} \\
Proceedings of the National Academy of Sciences, 112(3), 2015

\item Andrey D. Prjibelski, Irina Vasilinetc, Anton Bankevich, Alexey Gurevich, Tatiana Krivosheeva, \textbf{Sergey Nurk}, Son Pham, Anton Korobeynikov, Alla Lapidus and Pavel A. Pevzner\\
\textbf{``ExSPAnder: a universal repeat resolver for DNA fragment assembly''.} \\
Bioinformatics, 30(12), i293-i301, 2014

\item \textbf{Sergey Nurk}$^*$, Anton Bankevich$^*$ ($^*$equal contribution)%et al.\\
, Dmitry Antipov, Alexey A. Gurevich, Anton Korobeynikov, Alla Lapidus, Andrey D. Prjibelski, Alexey Pyshkin, Alexander Sirotkin, Yakov Sirotkin, Ramunas Stepanauskas, Scott R. Clingenpeel, Tanja Woyke, Jeffrey S. Mclean, Roger Lasken, Glenn Tesler, Max A. Alekseyev, and Pavel A. Pevzner \\
\textbf{``Assembling Single-Cell Genomes and Mini-Metagenomes From Chimeric MDA Products''.} \\
Journal of Computational Biology 20(10), 2013

%\item \textbf{Sergey Nurk}$^*$, Anton Bankevich$^*$ ($^*$equal contribution)%et al.\\
%, Dmitry Antipov, Alexey A. Gurevich, Anton Korobeynikov, Alla Lapidus, Andrey Prjibelsky, Alexey Pyshkin, Alexander Sirotkin, Yakov Sirotkin, Ramunas Stepanauskas, Jeffrey McLean, Roger Lasken, Scott R. Clingenpeel, Tanja Woyke, Glenn Tesler, Max A. Alekseyev, and Pavel A. Pevzner \\
%\textbf{``Assembling Genomes and Mini-metagenomes from Highly Chimeric Reads''.} \\
%Proceedings of the 17th Annual International Conference on Research in Computational Molecular Biology, Lecture Notes in Computer Science 7821 (2013).

\item Jeffrey S. McLean, Mary-Jane Lombardo, Michael G. Ziegler, Mark Novotny, Joyclyn Yee-Greenbaum, Jonathan H. Badger, Glenn Tesler, \textbf{Sergey Nurk}, Valery Lesin, Daniel Brami, Adam P. Hall, Anna Edlund, Lisa Z. Allen, Scott Durkin, Sharon Reed, Francesca Torriani, Kenneth H. Nealson, Pavel A. Pevzner, Robert Friedman, J. Craig Venter and Roger S. Lasken \\
\textbf{``Genome of the pathogen Porphyromonas gingivalis recovered from a biofilm in a hospital sink using a high-throughput single cell genomic platform''.} \\
Genome research, 23(5), 867-877

\item Anton Bankevich$^*$, \textbf{Sergey Nurk}$^*$ ($^*$equal contribution)%et al.\\
, Dmitry Antipov, Alexey Gurevich, Mikhail Dvorkin, Alexander Kulikov, Valery Lesin, Sergey Nikolenko, Son Pham, Andrey Prjibelski, Alexey Pyshkin, Alexander Sirotkin, Nikolay Vyahhi, Glenn Tesler, Max Alekseyev and Pavel Pevzner \\
\textbf{``SPAdes: a New Genome Assembler and its Applications to Single Cell Sequencing''.}\\
Journal of Computational Biology 19(5), 2012
%\end{innerlist}
%\end{enumerate}

%Miscellaneous
\item \textbf{Sergey Nurk} \\
\textbf{``An $O(2^{0.4058m})$ Upper Bound for \textit{Circuit SAT}''.}\\
PDMI preprint 10/2009, 2009

%\item
%\textbf{Sergey Nurk}, Pavel A. Pevzner\\
%\textbf{``SPArcle: Identifying Microbial Genome Variations with Colored de Bruijn Graphs''.} \\
%Manuscript in preparation

%\item
%\textbf{Sergey Nurk}, Yury Gorshkov and Pavel A. Pevzner\\
%\textbf{``Reconstructing individual genomes from multiple metagenomic samples with MTS pipeline''.}\\
%Manuscript in preparation
\end{outerlist}
%\end{enumerate}

